react/javascript/tile-visualization/.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
```

Comment:
not important but do you have plans to update to including semicolons?

Reply:
No plans as yet.